scrapers/nus-v2/src/tasks/GetSemesterData.ts

```diff
@@ -93,6 +94,10 @@ export function cleanModuleInfo(module: SemesterModule) {
     cleanedModule.title = titleize(cleanedModule.title);
   }
 
+  if (typeof cleanedModule.description !== 'undefined') {
```

Comment:
A bit shorter and more idiomatic way to check for null-like values 

```suggestion
  if (cleanedModule.description != null) {
```

Reply:
Actually why not just

```suggestion
  if (cleanedModule.description) {
```

since we don't need to care about empty strings as well

Reply:
Technically true, but I think `!= null` makes it clear this is a null check. But yeah it is functionally the same

Reply:
going with `!= null` in c0e6be4